podium-gateway/src/main/webapp/app/backoffice/modules/organisation/organisation.model.ts

```diff
@@ -17,13 +17,25 @@ export class Organisation {
     uuid?: string;
     name?: string;
     shortName?: string;
-    deleted?: boolean;
     activated?: boolean;
     organisationUuid?: string;
     public roles?: Role[];
     public requestTypes?: RequestType[];
 
-    constructor() {
```

Comment:
A bit worried about Object.assign since it is not supported in IE. Not knowing the status at the moment, do you have any idea? I can do another workaround tho